overlay.nix

```diff
@@ -0,0 +1,9 @@
+# You can use this file as a nixpkgs overlay. This is useful in the
+# case where you don't want to add the whole cbspkgs namespace to your
+# configuration.
+
+self: super:
+
+{
+  cbspkgs = import ./default.nix { pkgs = super; };
```

Comment:
I think someone brought this up yesterday: when we pass `pkgs` here, isn't there a mismatch of `nixpkgs` and `pkgs` in `default.nix`?

Reply:
nixpkgs is only used to define `pkgs` in `default.nix`. So it's either "their" nixpkgs or "ours".

Reply:
Could the line `inherit (niv) nixpkgs;` in `default.nix` could go away?

Reply:
that's not desirable, because we want to provide the user the nixpkgs that we pinned. This way users can select the *known working* version for packages that might be problematic and they can evaluate to expressions that are already binary-cached.

The usecase for this would be:

```nix
cbspkgs = import <cbspkgs> {};
pkgs = import cbspkgs.nixpkgs { overlays = cbspkgs.overlays; };
```